Library/Formula/procyon-decompiler.rb

```diff
@@ -0,0 +1,17 @@
+class ProcyonDecompiler < Formula
+  homepage "https://bitbucket.org/mstrobel/procyon/wiki/Java%20Decompiler"
```

Comment:
Could you please add a short description above this line using `desc "…"`? Thanks!